doc/man3/SSL_CTX_set_read_ahead.pod

```diff
@@ -39,6 +41,18 @@ SSL_CTX_get_read_head() and SSL_get_read_ahead() are undefined for DTLS. Setting
 B<read_ahead> can impact the behaviour of the SSL_pending() function
 (see L<SSL_pending(3)>).
 
+Since SSL_read() can return B<SSL_ERROR_WANT_READ> for non-application data
+records, and SSL_has_pending() can't tell the difference between processed and
+unprocessed data, it's recommended that if read ahead is turned on that
+B<SSL_MODE_AUTO_RETRY> is also turned on using SSL_CTX_set_mode().
```

Comment:
Since its the default, I guess this changes the recommendation to not turn it off.

Reply:
I'm having some mixed feelings about this. It's just a default, that happens to have changed between 1.1.0 and 1.1.1. But I'll change it.